infoschema/tables.go

```diff
@@ -89,6 +89,7 @@ const (
 	tableTiKVRegionPeers                    = "TIKV_REGION_PEERS"
 	tableTiDBServersInfo                    = "TIDB_SERVERS_INFO"
 	tableTiDBClusterInfo                    = "TIDB_CLUSTER_INFO"
+	tableTiFlashReplica                     = "TABLE_TIFLASH_REPLICA"
```

Comment:
```suggestion
	tableTiFlashReplica                     = "TIFLASH_REPLICA"
```
Please follow the previous naming styles.

Reply:
Done.